packages/paste-core/components/input-box/stories/inputbox.stories.tsx

```diff
@@ -30,6 +31,14 @@ const CommonStories: React.FC<Omit<InputBoxProps, 'children'>> = ({variant, ...p
       <InputBox variant={variant} insertAfter={<Insert />} insertBefore={<Insert />} {...props}>
         <Input />
       </InputBox>
+      <InputBox variant={variant} insertAfter={<Insert />} insertBefore={<Insert />} {...props}>
+        <Box display="flex" width="100%" position="relative">
+          <Input />
+          <InputChevronWrapper element="test">
```

Comment:
question: is this element attribute used anywhere? if it's not, I don't think it's necessary, especially since this isn't how we want people to use it.

Reply:
It's added because it's a required prop